code/nodes/featureCreation/normal/funcs.q

```diff
@@ -0,0 +1,64 @@
+\d .automl
+
+// The functionality contained in this file covers the required and optional utilities for
+// normal feature creation within the automated machine learning library.
+
+// @kind function
+// @category featureCreation
+// @fileoverview Used in the recursive application of functions to a kdb+ table 
+// @param feat {tab} The feature data as a table 
+// @param func {(lambda;str)} function to be applied to the table
+// return {table} with the desired transforms applied recursively
+featureCreation.normal.applyFunc:{[feat;func]
+  typ:type func;
+  func:$[-11h=typ;get func;100h=typ;func;.automl.featureCreation.normal.default];
```

Comment:
May be worth adding a comment over this to say that once Deanna's changes to tts/sigfeats are in then this function should use the same 'util.qpyFuncSearch' function being used in that PR s.t. python feature creation procedures could in theory also be produced